todo_app/templates/index.html

```diff
@@ -21,7 +21,12 @@ <h1 class="display-4">To-Do App</h1>
        <h2>Items</h2>
        <ul class="list-group mb-4">  
         {% for item in items %}
-          <li>{{item.title}} - {{item.status}}</li>
+          <li>{{item.name}} - {{item.status}}</li>
+          {% if item.status != "Done"%}
+          <form action="complete-item/{{item.id}}" method="post">  
+            <button>Complete</button>
+            </form>
+            {% endif %}    
```

Comment:
Really minor, but while HTML doesn't care about whitespace like Python does, it's still worth being careful because it makes it easier to see what's going on at a glance:
```suggestion
          {% if item.status != "Done"%}
            <form action="complete-item/{{item.id}}" method="post">  
              <button>Complete</button>
            </form>
          {% endif %}    
```